README.md

```diff
@@ -87,6 +87,25 @@ Viewing word clouds doesn't require login access. The URL of the view page is in
         font size, the higher the voice pitch.
         - **"Apply conventional weighting"**: When enabled, more frequently submitted words will be highlighted.
 
+### Simulate a Word Cloud
+
+You can find a Node.js script named updateAnswers.js in [src/scripts/](https://github.com/inclusive-design/inverted-wordles/tree/main/src/scripts/) directory to generate or update existing word cloud answers.
+
+**Steps**
+
+1. **Find a Specific Word Cloud's Question File name**:
+    - In this repository, you can find a specific word cloud's question data file under [src/_data](https://github.com/inclusive-design/inverted-wordles/tree/main/src/_data) directory; they have suffic of -question.
+2. **Get File Name of the Answers to the Word Cloud**:
+    - From the step 1, you can get the answers file by replacing the -question with -answers; for example, a word cloud that has a question file named xxxx-xxxx-question.json has an answers file named xxxx-xxxx-answers.json.
```

Comment:
Probably backtick `-question` and `-answers` to highlight? The same comment applies to occurrences of file names and similar cases.